docs/source/key_components/dashboard_visualization.rst

```diff
@@ -0,0 +1,109 @@
+Dashboard Visualization
```

Comment:
better to enrich the introduction of shown features/visualization

add charts as example if needed

Reply:
this part is added to docs/scenarios/cim.rst#example & docs/scenarios/citi_bike.rst#example

Reply:
Or add detailed introduction and example charts in scenario.rst, and add link to the scenario part.